plugins/orchestrator-backend/src/helpers/queryBuilder.ts

```diff
@@ -0,0 +1,37 @@
+import { Pagination } from '../types/pagination';
+
+export function buildGraphQlQuery(
+  type: string,
+  queryBody: string,
+  whereClause?: string,
+  pagination?: Pagination,
+): string {
```

Comment:
Let's put the function arguments in an object, please. It is less error-prone if we have named args, especially when there is more than one arg with the same type.

```suggestion
export function buildGraphQlQuery(args: {
  type: string;
  queryBody: string;
  whereClause?: string;
  pagination?: Pagination;
}): string {
```

Please also modify the rest of the code accordingly.

Reply:
[a8ae2e7](https://github.com/janus-idp/backstage-plugins/pull/1164/commits/a8ae2e7ab4f36782e0cc3d4f970562468fbbb0e6)